R/functions-MSnExp.R

```diff
@@ -377,7 +593,12 @@ plotXIC_MSnExp <- function(x, ...) {
                 immediate = TRUE, call = FALSE)
     ## Define the layout.
     dots <- list(...)
-    layout(.vertical_sub_layout(length(x)))
+    if (any(names(dots) == "layout")) {
```

Comment:
That's a fix to allow using a pre-defined plot layout and avoid that being overruled by the function.